internal/services/api-gateway/internal/proto/weather.proto

```
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package weather;
```

Buf (1.55.1)

Files with package "weather" must be within a directory "weather" relative to root but were in directory "internal/services/api-gateway/internal/proto".

(PACKAGE_DIRECTORY_MATCH)